### PR TITLE
DAOS-2216 dfs & duns: expose Chunks size as configurable parameter

### PR DESCRIPTION
**Pull request by mchaarawi**
- Add chunk size setting to dfs_open()
- Allow setting of chunk size through DUNS tool and set in extended attribute

Signed-off-by: Mohamad Chaarawi <mohamad.chaarawi@intel.com>

### CODE REVIEW

**Comment by mchaarawi** [comment]
both travis CI builds on ubuntu failed only because the build was taking too long.. not sure how to increase the build timeout on those..

**Comment by johannlombardi** [comment]
Could you please look into Travis' failure?

**Comment by johannlombardi** [comment]
ah, just a timeout on ubuntu:

gcc -o build/src/tests/suite/daos_epoch_io.o -c -Werror -g -Wshadow -Wall -Wno-missing-braces -fpic -D_GNU_SOURCE -DD_LOG_V2 -O2 -DDAOS_VERSION=\"0.0.2\" -Wframe-larger-than=4096 -
Isrc/include/daos -Isrc/include -Iinstall/include -I/usr/include -Isrc/tests/suite src/tests/suite/daos_epoch_io.c

The job exceeded the maximum time limit for jobs, and has been terminated.

File: src/include/daos_uns.h [hunk 1]
```diff
@@ -46,6 +46,8 @@ struct duns_attr_t {
 	daos_cont_layout_t	da_type;
 	/** Default Object Class for all objects in the container */
 	daos_oclass_id_t	da_oclass;
+	/** Default Chunks size for all files in container */
+	daos_size_t		da_chunk_size;
```
Comment (bfaccini) [inline src/include/daos_uns.h]: may be field name should also state default for visibility ?

Comment (mchaarawi) [reply src/include/daos_uns.h]: OK, will address in follow on patch (for chunk size and object class)

**Pull request merged**